app/models/miq_request_task/state_machine.rb

```diff
@@ -21,29 +21,15 @@ def signal(phase)
     begin
       send(phase)
     rescue => err
-      if [:finish, :provision_error].include?(phase.to_sym)
-        _log.error("[#{err.message}] encountered during [#{phase}]")
-        _log.log_backtrace(err)
+      case phase
+      when :finish
+        $log.error("[#{err.class}: #{err.message}] encountered during [#{phase}]")
```

Comment:
Please use _log over $log as it was before.  As such, there is no need for adding the class in

Reply:
Alright, updated.